package.json

```diff
@@ -51,11 +49,16 @@
     "ts-node": "10.9.1",
     "ts-proto": "1.155.1",
     "tsc-alias": "^1.8.7",
-    "typescript": "5.1.6",
-    "vitest": "^0.34.1"
```

Comment:
I remember I ran into an issue when porting these endpoints, unfortunately I forgot what it was 😅  I thought "What is `vitest` getting for us"? So I just replaced with standard jest since we aren't doing anything special in this repo.